lib/chef-acceptance/chef_runner.rb

```diff
@@ -13,12 +13,14 @@ class ChefRunner
     attr_reader :test_suite
     attr_reader :recipe
     attr_reader :duration
+    attr_reader :timeout
 
-    def initialize(test_suite, recipe)
+    def initialize(test_suite, recipe, timeout)
       @test_suite = test_suite
       @acceptance_cookbook = test_suite.acceptance_cookbook
       @recipe = recipe
       @duration = 0
+      @timeout = timeout
```

Comment:
Not a huge fan of this either because if not passed you'd have to set another default value.